studio/schemas/resource.js

```diff
@@ -0,0 +1,249 @@
+import {MdKitchen as icon} from 'react-icons/md'
+
+export default {
+  name: 'resource',
+  title: 'Resource',
+  type: 'document',
+  icon,
+  fields: [
+    {
+      name: 'type',
```

Comment:
type primarily describes how we handle it in the UI and what we expect in that regard.